plugins/database/mssql/mssql_test.go

```diff
@@ -357,6 +357,53 @@ func TestDeleteUser(t *testing.T) {
 	assertCredsDoNotExist(t, connURL, dbUser, initPassword)
 }
 
+func TestDeleteUserContainedDB(t *testing.T) {
+	cleanup, connURL := mssqlhelper.PrepareMSSQLTestContainer(t)
```

Comment:
@tvoran We talked about adding a cleanup helper function to contained db, however doesn't the `PrepareMSSQLTestContainer` function provide us a clean container each time we run this test so it shouldn't interfere with the happy path as well.

Reply:
Yep, I think that's correct. In the previous commit it looks like a single test container was being used for all the test cases, so that's why it was running into side effect issues.

Reply:
Sounds good!